pioneer/packages/joy-utils/src/transport/tokenomics.ts

```diff
@@ -341,4 +350,133 @@ export default class TokenomicsTransport extends BaseTransport {
       }
     };
   }
+
+  async networkStatistics () {
+    const blockHeight = (await this.api.derive.chain.bestNumber()).toNumber();
+    const numberOfMembers = (await this.api.query.members.nextMemberId() as MemberId).toNumber();
+    const content = await this.api.query.dataDirectory.knownContentIds() as Vec<ContentId>;
+    const numberOfChannels = (await this.api.query.contentWorkingGroup.nextChannelId() as ChannelId).toNumber();
```

Comment:
Just noticed this is a similar case to forum categories and posts (`1` needs to be substracted):
```
console.log((await api.query.contentWorkingGroup.nextChannelId()))
console.log((await api.query.contentWorkingGroup.channelById.keys()).length)
```